### PR TITLE
Merge CUDA support to racon

### PR DESCRIPTION
**Pull request by tijyojwad**
Adding support for CUDA accelerated polishing using `cudapoa` module in ClaraGenomicsAnalysis SDK.

### CODE REVIEW

**Comment by tijyojwad** [comment]
Hey @rvaser, did you get a chance to look through this PR?

**Comment by rvaser** [comment]
Tomorrow for sure! Sorry for the delay :(

**Comment by tijyojwad** [comment]
No problem, @rvaser ! Thanks :)

File: src/window.hpp [hunk 1]
```diff
@@ -24,11 +24,12 @@ enum class WindowType {
 };
 
 class Window;
-std::unique_ptr<Window> createWindow(uint64_t id, uint32_t rank, WindowType type,
+std::shared_ptr<Window> createWindow(uint64_t id, uint32_t rank, WindowType type,
```
Comment (rvaser) [inline src/window.hpp]: Any particular reason for changing `std::unique_ptr` to `std::shared_ptr`?

Comment (rvaser) [reply src/window.hpp]: Nvm, found what I was looking for :)

**Pull request merged**